tool/test-bundled-gems.rb

```diff
@@ -33,7 +35,7 @@
   next if bundled_gems&.none? {|pat| File.fnmatch?(pat, gem)}
   next unless File.directory?("#{gem_dir}/src/#{gem}/test")
 
-  test_command = [ruby, "-C", "#{gem_dir}/src/#{gem}", rake, "test"]
+  test_command = [ruby, *run_opts, "-C", "#{gem_dir}/src/#{gem}", rake, "test"].compact
```

Comment:
You shouldn't need `.compact` for `*nil`.

```suggestion
  test_command = [ruby, *run_opts, "-C", "#{gem_dir}/src/#{gem}", rake, "test"]
```